### ISSUE TITLE
Attempting to create blueprint subclass from Cesium3DTileset results in fatal error

### ISSUE THREAD

**Issue by argallegos** [CesiumGS/cesium-unreal#336]
As the title suggests, creating a subclass using ACesium3DTileset as a parent causes an instant crash. This is replicable by right-clicking in the content browser and creating a new blueprint using ACesium3DTileset as the base, or by selecting a 3D tileset in the World Outliner and hitting the "Blueprint/Add Script" button.

It crashes so quickly it doesn't even catch an error, unless you're running it through Visual studio, in which case I got the following. 

```
[2021.04.13-14.31.11:780][102]LogSlate: Window 'Pick Parent Class' being destroyed
[2021.04.13-14.31.11:787][102]LogContentBrowser: Deferred new asset file creation: NewBlueprint
[2021.04.13-14.31.11:797][102]LogContentBrowser: Creating deferred item: NewBlueprint
[2021.04.13-14.31.11:850][105]LogRenderer: Reallocating scene render targets to support 908x488 Format 10 NumSamples 1 (Frame:142).
The thread 0x2eb4 has exited with code 0 (0x0).
[2021.04.13-14.31.22:984][609]LogContentBrowser: Attempting asset rename: NewBlueprint -> EditedTilesetSubclass
[2021.04.13-14.31.22:994][609]LogContentBrowser: End creating deferred item NewBlueprint
[2021.04.13-14.31.24:918][681]LogAssetEditorSubsystem: Opening Asset editor for Blueprint /Game/EditedTilesetSubclass.EditedTilesetSubclass
[2021.04.13-14.31.25:250][681]LogNativeClassHierarchy: Verbose: Native class hierarchy updated for 'BlueprintGraph' in 0.0006 seconds. Added 126 classes and 0 folders.
[2021.04.13-14.31.26:266][681]LogCesium: Loading tileset for asset ID 0
[2021.04.13-14.31.26:267][681]LogCesium: Loading tileset for asset ID 0 done
Exception thrown at 0x00007FF8A02CD759 in UE4Editor.exe: Microsoft C++ exception: std::invalid_argument at memory location 0x0000001A48D789D0.
Unhandled exception at 0x00007FF8A01B286E (ucrtbase.dll) in UE4Editor.exe: Fatal program exit requested.
```

This is the code that seems to be causing the issue, in Cesium3DTileset.cpp. 

![image](https://user-images.githubusercontent.com/39537389/114572886-e4c93480-9c45-11eb-9fc9-48d18700fc65.png)

**Comment by javagl** [comment]
I'm pretty sure this will be "fixed" via https://github.com/CesiumGS/cesium-unreal/pull/329/files#diff-538e6be1390d243ee2267e9be89a07dd4ff067752f3894d5fd0ed9515fe0dc68L11 , but the exact behavior has to be tested.

The chain of events leading to this crash is described in https://github.com/CesiumGS/cesium-unreal/issues/290#issuecomment-817360101 

I've opened https://github.com/CesiumGS/cesium-native/issues/205 accordingly.

**Comment by kring** [comment]
@argallegos can you confirm if this was fixed by #329?

**Comment by kring** [comment]
I just tried creating a new blueprint class derived from ACesium3DTileset, and it works fine. Closing.
